[2025-01-15T18:19:17.073+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: ETL_influencer_marchendise.data_load manual__2025-01-15T18:19:01.292437+00:00 [queued]>
[2025-01-15T18:19:17.079+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: ETL_influencer_marchendise.data_load manual__2025-01-15T18:19:01.292437+00:00 [queued]>
[2025-01-15T18:19:17.079+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2025-01-15T18:19:17.079+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 2
[2025-01-15T18:19:17.079+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2025-01-15T18:19:17.086+0000] {taskinstance.py:1389} INFO - Executing <Task(BashOperator): data_load> on 2025-01-15 18:19:01.292437+00:00
[2025-01-15T18:19:17.088+0000] {standard_task_runner.py:52} INFO - Started process 612 to run task
[2025-01-15T18:19:17.090+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'ETL_influencer_marchendise', 'data_load', 'manual__2025-01-15T18:19:01.292437+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/P2M3_afif_makruf_DAG.py', '--cfg-path', '/tmp/tmpshjkmsi_', '--error-file', '/tmp/tmp_ae1vu5h']
[2025-01-15T18:19:17.091+0000] {standard_task_runner.py:80} INFO - Job 4: Subtask data_load
[2025-01-15T18:19:17.122+0000] {task_command.py:371} INFO - Running <TaskInstance: ETL_influencer_marchendise.data_load manual__2025-01-15T18:19:01.292437+00:00 [running]> on host 2de0c6755ca5
[2025-01-15T18:19:17.156+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=afif
AIRFLOW_CTX_DAG_ID=ETL_influencer_marchendise
AIRFLOW_CTX_TASK_ID=data_load
AIRFLOW_CTX_EXECUTION_DATE=2025-01-15T18:19:01.292437+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2025-01-15T18:19:01.292437+00:00
[2025-01-15T18:19:17.156+0000] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2025-01-15T18:19:17.157+0000] {subprocess.py:74} INFO - Running command: ['/bin/bash', '-c', 'sudo -u *** python /opt/***/dags/load.py']
[2025-01-15T18:19:17.161+0000] {subprocess.py:85} INFO - Output:
[2025-01-15T18:19:26.345+0000] {subprocess.py:96} INFO - Command exited with return code 0
[2025-01-15T18:19:26.382+0000] {taskinstance.py:1412} INFO - Marking task as SUCCESS. dag_id=ETL_influencer_marchendise, task_id=data_load, execution_date=20250115T181901, start_date=20250115T181917, end_date=20250115T181926
[2025-01-15T18:19:26.436+0000] {local_task_job.py:156} INFO - Task exited with return code 0
[2025-01-15T18:19:26.467+0000] {local_task_job.py:279} INFO - 0 downstream tasks scheduled from follow-on schedule check
